### PR TITLE
EXPOSERS: Put Team

### LINKED ISSUE #125: EXPOSERS: Put `Team`

### PR DESCRIPTION
**Pull request by DadabaevAlisher**
closes #125

### CODE REVIEW

**Review by sarvarbek-dev — COMMENTED** [review]
LGTM

**Review by sarvarbek-dev — COMMENTED** [review]
LGTM

**Review by sarvarbek-dev — APPROVED** [review]

File: Tarteeb.Api/Controllers/TeamsController.cs [hunk 1]
```diff
@@ -98,5 +98,40 @@ public async ValueTask<ActionResult<Team>> GetTeamByIdAsync(Guid Id)
...
+            catch (TeamValidationException teamValidationException)
+            {
+                return BadRequest(teamValidationException.InnerException);
+            }
+            catch (TeamDependencyValidationException teamDependencyValidationException)
+                when (teamDependencyValidationException.InnerException is AlreadyExistsTeamException)
+            {
+                return Conflict(teamDependencyValidationException.InnerException);
+            }
+            catch (TeamDependencyException teamDependencyException)
```
Comment (jasurbekyusuf) [inline Tarteeb.Api/Controllers/TeamsController.cs:127]: InnerException qaytarishi kerak

**Review by jasurbekyusuf — CHANGES_REQUESTED** [review]

File: Tarteeb.Api/Controllers/TeamsController.cs [hunk 2]
```diff
@@ -98,5 +98,40 @@ public async ValueTask<ActionResult<Team>> GetTeamByIdAsync(Guid Id)
                 return InternalServerError(teamServiceException.InnerException);
             }
         }
+
+        [HttpPut]
+
```
Comment (Javlonbek-dev) [inline Tarteeb.Api/Controllers/TeamsController.cs:103]: Remove Line

**Comment by Javlonbek-dev** [comment]
@DadabaevAlisher  Resolve conflicts

File: Tarteeb.Api/Controllers/TeamsController.cs [hunk 3]
```diff
@@ -119,21 +119,17 @@ public async ValueTask<ActionResult<Team>> DeleteTeamByIdAsync(Guid teamId)
...
             }
             catch (TeamDependencyException teamDependencyException)
             {
                 return InternalServerError(teamDependencyException.InnerException);
             }
             catch (TeamServiceException teamServiceException)
             {
-                return InternalServerError(teamServiceException.InnerException);
+                return InternalServerError(teamServiceException);
             }
```
Comment (jasurbekyusuf) [inline Tarteeb.Api/Controllers/TeamsController.cs:133]: return InternalServerError(teamServiceException) Change To `return InternalServerError(teamServiceException.InnerException)`

**Review by DadabaevAlisher — COMMENTED** [review]
CODE RUB: Add InnerException

**Review by jasurbekyusuf — COMMENTED** [review]
LGTM

**Review by jasurbekyusuf — APPROVED** [review]

**Pull request merged**